src/cleanvision/imagelab.py

```diff
@@ -1,3 +1,9 @@
+"""
```

Comment:
High-level feedback: Use numpy format for docstrings.
Argument names should be enclosed in single backticks: `x`.
Code snippets should be enclosed in double backticks: ``method(...)``. 

When referencing classes/methods that are part of CleanVision, it would be nice to link to them. How to do this is described here:
https://github.com/cleanlab/cleanlab/blob/master/DEVELOPMENT.md#documentation-style